scipy/optimize/tests/test_optimize.py

```diff
@@ -1175,6 +1175,11 @@ def test_fminbound_scalar(self):
         x = optimize.fminbound(self.fun, 1, np.array(5))
         assert_allclose(x, self.solution, atol=1e-6)
 
+    def test_gh11207(self):
+        def fun(x):
+            return x**2
+        optimize.fminbound(fun, 0, 0)
```

Comment:
Is it worth checking that the minimum is also near 0? I assume that's the case between `0` and `0`, though it might be stranger than that I suppose.